docker-compose.yml

```diff
@@ -70,16 +70,18 @@ services:
       KSQL_CONFIG_DIR: "/etc/ksql"
       KSQL_LOG4J_OPTS: >-
         -Dlog4j.configuration=file:/etc/ksql/log4j-rolling.properties
-      KSQL_BOOTSTRAP_SERVERS: "kafka:9092"
       KSQL_HOST_NAME: ksql-server
       KSQL_APPLICATION_ID: "cp-all-in-one"
       KSQL_LISTENERS: "http://ksql-server:8088"
       KSQL_CACHE_MAX_BYTES_BUFFERING: 0
-      KSQL_KSQL_SCHEMA_REGISTRY_URL: "http://schema-registry:8081"
       KSQL_PRODUCER_INTERCEPTOR_CLASSES: >-
         io.confluent.monitoring.clients.interceptor.MonitoringProducerInterceptor
       KSQL_CONSUMER_INTERCEPTOR_CLASSES: >-
         io.confluent.monitoring.clients.interceptor.MonitoringConsumerInterceptor
+    volumes:
+      - ./.secrets:/.secrets
```

Comment:
Can/should we use the "secrets" feature of docker-compose for this? See https://docs.docker.com/compose/compose-file/#secrets